src/main/java/frc/robot/commands/OnBridgeUntilMovement.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Constants.DriveConstants;
+import frc.robot.Robot;
+import frc.robot.util.MotionHandler;
+import frc.robot.util.MotionHandler.MotionMode;
+
+public class OnBridgeUntilMovement extends SequentialCommandGroup {
+  double robotSpeed = 0;
+  double backSpeed = 0;
+
+  public OnBridgeUntilMovement(boolean gridside) {
+    robotSpeed = 2 * (gridside ? 1 : -1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
+    backSpeed = 0.4 * (gridside ? -1 : 1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
```

Comment:
Can  we simplify this to something like
```
if (condition1 || condition 2) { robotSpeed = ...; } else { robotSpeed = -...; }
```
and the same thing for back speed?

Will be a bit easier to debug.